app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      def index
+        @books = Book.all
+        render_paginated json: @books
```

Comment:
It is not necessary the `json` type format on response, because render_paginated put data on JSON format and this can be made in one line, like this:
```suggestion
        render_paginated Book.all
```

Reply:
Done. Great tip !